src/client/sandbox/native-methods.ts

```diff
@@ -508,9 +512,9 @@ class NativeMethods {
         this.getAttributeNS                = nativeElement.getAttributeNS;
         this.insertAdjacentHTML            = nativeElement.insertAdjacentHTML;
         this.insertBefore                  = nativeElement.insertBefore;
-        this.insertCell                    = createElement('tr').insertCell;
-        this.insertTableRow                = createElement('table').insertRow;
-        this.insertTBodyRow                = createElement('tbody').insertRow;
+        this.insertCell                    = (<HTMLTableRowElement>createElement('tr')).insertCell;
```

Comment:
It's better to always use `as` to convert types, you'll have to enable a TSLint rule that forces it at some point.